common/src/main/java/whocraft/tardis_refined/common/world/chunk/TardisChunkGenerator.java

```diff
@@ -37,12 +40,109 @@ public class TardisChunkGenerator extends ChunkGenerator {
             instance.group(RegistryOps.retrieveElement(ChunkGenerators.TARDIS_BIOME)).apply(instance,
                     instance.stable(TardisChunkGenerator::new)));
 
+
+    public final Random random;
+
+    // Some parameter values.
+    public final int distanceBetweenGravityWell = 18;
+    public final int arsChunkSize = ARSStructurePiece.LOCKED_PIECE_CHUNK_SIZE;
+    private final int chunkSize = 16;
+
+
+
     public TardisChunkGenerator(Holder<Biome> holder) {
         super(new FixedBiomeSource(holder));
+        this.random = new Random();
     }
 
     @Override
     public void applyBiomeDecoration(WorldGenLevel pLevel, ChunkAccess pChunk, StructureManager pStructureManager) {
+
+        if (pChunk.getPos().x == 63  && pChunk.getPos().z == 0) {
+            pLevel.getLevel().getStructureManager().get(new ResourceLocation(TardisRefined.MODID, "corridors/corridor_hub/corridor_hub_center")).ifPresent(structure -> {
+                int height = 97 - 21;
```

Comment:
Don't execute unnecessary math, put the result

Reply:
No longer executing unnecessary math.